### DISCUSSION TITLE
Use the numeric keys to select mod presets

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#19683] [Ideas] [answered]
Since the number keys are no longer used (by default) by the mod select overlay, I suggest using them to select the mod presets. You can only take advantage of this feature for the first nine presets (or even 10 if we count the number 0).

**Comment by peppy** [comment, marked as answer]
Seems like a good idea, but I would also propose we add some kind of global hotkeys for selecting presets when the mod select overlay is not open. Even if there's only one "Toggle last used mod preset".
